src/databases/WData/wdata.c

```diff
@@ -0,0 +1,1056 @@
+/**
+ * W-SLDA Toolkit
+ * This file is taken from repository
+ * https://gitlab.fizyka.pw.edu.pl/wtools/wdata
+ * which provides C implementation of WData format under GNU GPLv3 license. 
+ * */
+
+/*****************************************************************************
+// Copyright (c) Lawrence Livermore National Security, LLC and other VisIt
+// Project developers.  See the top-level LICENSE file for dates and other
+// details.  No copyright assignment is required to contribute to VisIt.
+*****************************************************************************/
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
```

Comment:
There will be a problem compiling on Windows due to use of unistd and libgen headers.
I should be able to make the necessary changes after the merge of this PR if you don't have access to Windows machines yourself. (We use Visual Studio to compile VisIt on Windows).

Reply:
I did not want to be rude, it's my first PR and this experience is totally new for me.
No one in our group has access to a Windows machine and Your help would be very helpful

Reply:
It is not rude at all, I understand not everyone has access to Windows.